content/en/tools/digital-currencies/defining/creating-token-definitions.md

```diff
@@ -0,0 +1,163 @@
+---
+date: '2023-03-21'
+lastmod: '2023-03-21'
+section_menu: tools
+menu:
+  tools:
+    name: "Creating Token Definitions"
+    weight: 200
+    parent: digital-currencies-token-definitions
+    identifier: digital-currencies-creating-token-definitions
+    description: "Digital Currencies documentation describing how to create token definitions via the GUI"
+title: "Creating Token Definitions"
+---
+
```

Comment:
```suggestion
## Creating Token Definitions
```

Reply:
As above, don't think we need an initial heading as that automatically comes from the Title of the page. Let me know if this is otherwise.